jet_tracking/devices.py

```diff
@@ -664,7 +664,7 @@ def __init__(self, ROI_port,
         self.ROI_image.enable.put('Enabled')
 
 
-class Questar(PCDSDetector):
+class Questar(PCDSAreaDetector):
```

Comment:
I would like to see the `Questar` device changed to `Inline` or `OnAxis`, since this generally encapsulates the jet tracking setup better than referring to CXI hardware.